# ff does not recognize aarch64 as a valid architecture.
# Patch by Terje Kvernes <terjekv@math.uio.no>
# Pushed upstream: https://github.com/truecluster/ff/pull/4
diff --git a/src/utk_platform_macros.hpp b/src/utk_platform_macros.hpp
index d82ad4f..027b685 100644
--- a/src/utk_platform_macros.hpp
+++ b/src/utk_platform_macros.hpp
@@ -134,6 +134,8 @@
     #define UTK__Arch_ARM_ARM
   #elif defined(__arm__) && defined(__thumb__)
     #define UTK__Arch_ARM_THUMB
+  #elif defined(__aarch64__)
+    #define UTK__Aarch_AARCH64
   #elif defined(__sh__)
     #define UTK__Arch_SuperH
   #else
